jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/bpmn2/BPMN2DataServiceImpl.java

```diff
@@ -159,6 +159,9 @@ private ProcessAssetDesc fillProcessDefinition(ProcessDescriptor helper, KieCont
 
         definition.setAssociatedEntities(helper.getTaskAssignments());
         definition.setProcessVariables(helper.getInputs());
+        for(String varName : helper.getInputs().keySet()) {
+            definition.addTagsForVariable(varName, definition.getTagsForVariable(varName));
```

Comment:
I believe we should call `helper.getInputsTags()` instead here
```suggestion
            definition.addTagsForVariable(varName, helper.getInputTags(varName));
```

Reply:
it is not the same thing.